internal/manager/controllerdef.go

```diff
@@ -263,6 +263,7 @@ func setupControllers(
 				Scheme:           mgr.GetScheme(),
 				DataplaneClient:  dataplaneClient,
 				CacheSyncTimeout: c.CacheSyncTimeout,
+				StatusQueue:      kubernetesStatusQueue,
```

Comment:
StatusQueue has to be added as a dependency to every controller listening to the in-memory events.